ArcGISRuntimeSDKQt_CppSamples/Geometry/SpatialOperations/SpatialOperations.cpp

```diff
@@ -0,0 +1,135 @@
+// [WriteFile Name=SpatialOperations, Category=Geometry]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "SpatialOperations.h"
+
+#include "Map.h"
+#include "MapQuickView.h"
+#include "Basemap.h"
+#include "Point.h"
+#include "Part.h"
+#include "PolygonBuilder.h"
+#include "SimpleFillSymbol.h"
+#include "SimpleLineSymbol.h"
+#include "GeometryEngine.h"
+
+using namespace Esri::ArcGISRuntime;
+
+SpatialOperations::SpatialOperations(QQuickItem* parent /* = nullptr */):
+  QQuickItem(parent),
+  m_polygon1(Geometry()),
+  m_polygon2(Geometry()),
+  m_inputsOverlay (new GraphicsOverlay()),
+  m_outputsOverlay (new GraphicsOverlay())
```

Comment:
The GraphicsOverlays will need parents, or need to be deleted in the destructor.